android/build.gradle

```diff
@@ -13,7 +13,16 @@ buildscript {
     }
 }
 
+def REACT_NATIVE_VERSION = new File(['node', '--print',"JSON.parse(require('fs').readFileSync(require.resolve('react-native/package.json'), 'utf-8')).version"].execute(null, rootDir).text.trim())
+
 allprojects {
+    configurations.all {
+        resolutionStrategy {
+            // Remove this override in 0.65+, as a proper fix is included in react-native itself.
+            force "com.facebook.react:react-native:" + REACT_NATIVE_VERSION
+        }
```

Comment:
I've seen this workaround, and it works well for applications projects - My only minor concern is this is a library project, and this closure is not conditionally scoped. Will this also affect consuming application projects? If not, then good to merge :+1:

Reply:
it shouldn't affect the consuming projects.

Reply:
Sounds good, thanks!